internal/fourslash/baselineutil.go

```diff
@@ -181,7 +182,8 @@ func (f *FourslashTest) getBaselineContentForFile(
 		if options.startMarkerPrefix != nil {
 			if startPrefix := options.startMarkerPrefix(span.Location); startPrefix != "" {
 				if fileName == options.marker.FileName() && span.Range.Start == options.marker.LSPosition {
-					// ts.Debug.assert(!detailPrefixes.has(details[0]), "Expected only single prefix at marker location");
+					_, ok := detailPrefixes[*details[0]]
+					debug.Assert(!ok, "Expected only single prefix at marker location")
```

Comment:
I suspect the ones in this file should actually be `t.Error`; it used to be that in Strada we had a _couple_ debug asserts that should have been `chai` checks but were not.

Reply:
Interesting, most `FourslashTest` take `t` as their first arg, but I guess we don't have that here.

I'm surprised we don't have it as a plain field at this point. Only one is ever allocated per test.